command/status.go

```diff
@@ -103,14 +103,18 @@ func (c *StatusCommand) Run(args []string) int {
 	var match contexts.Context
 	matchCount := 0
 	for ctx, vers := range res.Matches {
-		if len(vers) == 1 {
+		if l := len(vers); l == 1 {
 			match = ctx
 			matchCount++
+		} else if l > 0 && vers[0] == id {
```

Comment:
Is it guaranteed that the first element would be the exact match, rather than vers[1] or later?

Reply:
Yeah they are sorted